README.md

```diff
@@ -16,7 +16,7 @@ projects branches.
 As an example, the `containerized-tests` Jenkins job consists out of the
 following files:
 
-- `containerized-tests.yaml` is a [Jenkins Job Builder](jjb) configuration
+- `jobs/containerized-tests.yaml` is a [Jenkins Job Builder](jjb) configuration
```

Comment:
[Jenkins Job Builder](jjb) is not linked to any file when I open the link its throwing 404 error, this needs to be changed isn't it

Reply:
Oh, indeed! The linking is broken, it is `[title](id)` now, but should be `[title][id]` instead. I'll send an additional commit to this PR to fix it.